### PR TITLE
redesign pixiv design

### PR DESCRIPTION
**Pull request by ghostdeleted**

### CODE REVIEW

File: utils/pixiv.py [hunk 1]
```diff
@@ -69,12 +70,7 @@ def shuffle_image_url(url: str):
 
 
 def recompile_date(date):
```
Comment (kijk2869) [inline utils/pixiv.py:72]:
```suggestion
def recompile_date(date: str) -> str:
```

File: utils/pixiv.py [hunk 2]
```diff
@@ -69,12 +70,7 @@ def shuffle_image_url(url: str):
...
-    regex = re.compile(r"(\d{4})-(\d{2})-(\d{2})")
-    match = regex.search(date)
-    year = match.group(1)
-    month = match.group(2)
-    day = match.group(3)
-    return f"{year}년 {month}월 {day}일"
```
Comment (SaidBySolo) [inline utils/pixiv.py:77]: why delete this?

**Review by SaidBySolo — CHANGES_REQUESTED** [review]

File: utils/pixiv.py [hunk 3]
```diff
@@ -69,12 +70,7 @@ def shuffle_image_url(url: str):
...
+    return datetime.datetime.strptime(date, "%Y-%m-%d").strftime("%Y년 %m월 %d일")
 
 
 async def request(method, endpoint, json=None):
```
Comment (kijk2869) [inline utils/pixiv.py:75]:
```suggestion
async def request(method: str, endpoint: str, json: Dict[str, str]=None) -> Dict[str, Any]:
```

File: utils/pixiv.py [hunk 4]
```diff
@@ -1,6 +1,7 @@
 import asyncio
 import re
 import time
+import datetime
```
Comment (kijk2869) [inline utils/pixiv.py:5]:
```suggestion
from typing import Dict, Any
```

**Review by kijk2869 — CHANGES_REQUESTED** [review]

File: utils/pixiv.py (hunk 2 above)
Comment (ghostdeleted) [reply utils/pixiv.py:77]: same code

File: utils/pixiv.py [hunk 5]
```diff
@@ -69,12 +70,7 @@ def shuffle_image_url(url: str):
 
 
 def recompile_date(date):
-    regex = re.compile(r"(\d{4})-(\d{2})-(\d{2})")
-    match = regex.search(date)
-    year = match.group(1)
-    month = match.group(2)
-    day = match.group(3)
-    return f"{year}년 {month}월 {day}일"
+    return datetime.datetime.strptime(date, "%Y-%m-%d").strftime("%Y년 %m월 %d일")
```
Comment (wonderlandpark) [inline utils/pixiv.py:73]:
```suggestion
    return datetime.datetime.strftime(date, "%Y년 %m월 %d일")
```

**Review by wonderlandpark — CHANGES_REQUESTED** [review]

**Review by wonderlandpark — CHANGES_REQUESTED** [review]
Sannoob

**Comment by ghostdeleted** [comment]
> Sannoob

?

File: utils/pixiv.py [hunk 6]
```diff
@@ -76,7 +76,6 @@ def recompile_date(date):
     day = match.group(3)
     return f"{year}년 {month}월 {day}일"
```
Comment (kijk2869) [inline utils/pixiv.py]:
```suggestion
    return datetime.datetime.strptime(date, "%Y-%m-%dT%H:%M:%S%z").strftime("%Y년 %m월 %d일")
```

**Review by kijk2869 — CHANGES_REQUESTED** [review]

File: utils/pixiv.py (hunk 6 above)
Comment (kijk2869) [reply utils/pixiv.py]: Do not commit this suggestion.

Have to remove upper lines to commit this.

**Review by kijk2869 — APPROVED** [review]

**Review by SaidBySolo — APPROVED** [review]

**Comment by SaidBySolo** [comment]
merge into #79

**Pull request merged**